examples/conformer_energies/conformer_energies.ipynb

```diff
@@ -17,60 +31,111 @@
     "from openff.interchange.drivers import get_gromacs_energies, get_openmm_energies"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Choose the molecule we want by specifying a SMILES:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "SMILES = 10 * \"C\"  #  \"c1n(CCO)c(C(F)(F)(F))cc1CNCCl\""
```

Comment:
I'd be fine to just use the interesting-looking molecule - I bet using decane was just carry-over from some debugging